src/go/k8s/pkg/resources/statefulset.go

```diff
@@ -342,7 +342,6 @@ func (r *StatefulSetResource) obj() (k8sclient.Object, error) {
 								// sometimes a little bit of memory is consumed by other processes than seastar
 								"--reserve-memory " + redpandav1alpha1.ReserveMemoryString,
 								r.portsConfiguration(),
-								"--",
```

Comment:
NIT: The commit or the cover letter could point why we need to drop this flag.

The PR https://github.com/vectorizedio/redpanda/pull/1065